src/components/TradeWidget/TxMessage.tsx

```diff
@@ -1,21 +1,84 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import { formatTimeInHours } from 'utils'
 import { useFormContext } from 'react-hook-form'
 import { TokenDetails } from 'types'
 import styled from 'styled-components'
 import { TradeFormData } from '.'
 import { displayTokenSymbolOrLink } from 'utils/display'
+import { usePriceEstimationInOwl } from 'hooks/usePriceEstimation'
+import BigNumber from 'bignumber.js'
+import { ZERO_BIG_NUMBER } from 'const'
+
+import alertIcon from 'assets/img/alert.svg'
 
 interface TxMessageProps {
   sellToken: TokenDetails
   receiveToken: TokenDetails
+  networkId: number
 }
 
 const TxMessageWrapper = styled.div`
   padding: 1em;
 `
 
-export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken }) => {
+const Warning = styled.p`
+  background: beige;
+  padding: 0.5em;
+  box-shadow: inset 0 0 7px 3px #cc262647;
+  border-radius: 0.3em;
+  display: flex;
+
+  .alert {
+    min-width: 2.5em;
+  }
+`
+
+const MIN_ECONOMICAL_VIABLE_FEE_IN_OWL = 1.2
```

Comment:
If you wanna make it more "sophisticated", then you are querying the current gas price for a fast transaction and then calculate this number `MIN_ECONOMICAL_VIABLE_FEE_IN_OWL`

While this would still not be perfect, as the gas prices can change quickly, it might be a little bit better .
If you think that it is over-engineered, I am also fine with the current version :)